ext/opentelemetry-ext-docker-tests/tests/mysql/test_mysql_functional.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import unittest
+
+import mysql.connector
+
+from opentelemetry import trace as trace_api
+from opentelemetry.ext.mysql import trace_integration
+from opentelemetry.sdk.trace import Tracer, TracerProvider
+from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+    InMemorySpanExporter,
+)
+
+MYSQL_USER = os.getenv("MYSQL_USER ", "testuser")
+MYSQL_PASSWORD = os.getenv("MYSQL_PASSWORD ", "testpassword")
+MYSQL_HOST = os.getenv("MYSQL_HOST ", "localhost")
+MYSQL_PORT = int(os.getenv("MYSQL_PORT ", "3306"))
+MYSQL_DB_NAME = os.getenv("MYSQL_DB_NAME ", "opentelemetry-tests")
+
+
+class TestFunctionalMysql(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls._tracer_provider = TracerProvider()
+        cls._tracer = Tracer(cls._tracer_provider, None)
+        cls._span_exporter = InMemorySpanExporter()
+        cls._span_processor = SimpleExportSpanProcessor(cls._span_exporter)
+        cls._tracer_provider.add_span_processor(cls._span_processor)
+        trace_integration(cls._tracer)
+        # Try to connect to DB
+        for i in range(5):
+            try:
+                cls._connection = mysql.connector.connect(
+                    user=MYSQL_USER,
+                    password=MYSQL_PASSWORD,
+                    host=MYSQL_HOST,
+                    port=MYSQL_PORT,
+                    database=MYSQL_DB_NAME,
+                )
+                cls._cursor = cls._connection.cursor()
+                break
+            except mysql.connector.Error as err:
+                print(err)
+            time.sleep(5)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._connection.close()
+
+    def setUp(self):
+        self._span_exporter.clear()
+
+    def validate_spans(self):
+        spans = self._span_exporter.get_finished_spans()
+        self.assertEqual(len(spans), 2)
+        for span in spans:
+            if span.name == "rootSpan":
```

Comment:
Probably you want to test (with an `assert`) that there is one and only one `span` whose `name` attribute is `"rootSpan"`, because if it is not your test case will fail with a `NameError` later.